### PR TITLE
Spell check strings and comments

### PR DESCRIPTION
**Pull request by jwhonce**
Signed-off-by: Jhon Honce <jhonce@redhat.com>

### CODE REVIEW

File: pkg/secrets/secrets.go [hunk 1]
```diff
@@ -237,7 +237,7 @@ func addSecretsFromMountsFile(filePath, mountLabel, containerWorkingDir, mountPr
 	return mounts, nil
 }
 
-// addFIPSModeSecret creates /run/secrets/system-fips in the container
+// addFIPSsModeSecret creates /run/secrets/system-fips in the container
```
Comment (mheon) [inline pkg/secrets/secrets.go]: This looks like a function name, and the original looks correct

Comment (jwhonce) [reply pkg/secrets/secrets.go]: @mheon I made the comment follow the function name. I didn't try to break anything. :)

Comment (mheon) [reply pkg/secrets/secrets.go]: @jwhonce Could I convince you to rename the function to match the original comment, then :smile: 

It looks like a typo in the actual code the way it is now

**Comment by jwhonce** [comment]
@rh-atomic-bot retry

**Comment by rhatdan** [comment]
This is commical on how many of these there was. @jwhonce Thanks for the cleanup.
@rh-atomic-bot r+
We can change the interface call in a separate PR.

**Pull request closed without merging**